### DISCUSSION TITLE
Updating LTS kernel for kirkstone

### DISCUSSION THREAD

**Discussion by mnme** [Freescale/meta-freescale#1824] [Q&A] [answered]
I [prepared an update](https://github.com/mnme/meta-freescale/commit/41371665a996af3ceec00b4fb592ee616155deb9) for the kirkstone `linux-fslc-lts` kernel and am looking to contribute it back. I'd have opened a PR directly to the kirkstone branch, [as I did in the past](https://github.com/Freescale/meta-freescale/pull/949), but I noticed that you only take backports that go through master first. Is this correct? If yes, how can this recipe be upgraded as it doesn't exist in master anymore?

**Comment by otavio** [comment, marked as answer]
It usually goes through `master,` but there are exceptions, so please feel free to send the pull request.
